_includes/v22.2/ui/insights.md

```diff
@@ -0,0 +1,224 @@
+
+<img src="{{ 'images/v22.2/schema_insight.png' | relative_url }}" alt="Schema insight" style="border:1px solid #eee;max-width:100%" />
+{% endif -%}
+
+CockroachDB uses the threshold of 6 executions before offering an insight because it assumes that you are no longer merely experimenting with a query at that point.
+
+- **Insights:** Contains one of the following insight types: **Create Index**, **Alter Index**, **Replace Index**, **Drop Unused Index**.
+- **Details:** Details for each insight. Different insight types display different details fields:
+
+    - **Create Index**, **Alter Index**, or **Replace Index**: A **Statement Fingerprint** field displays the statement fingerprint that would be optimized with the creation, alteration, or replacement of the index; and a **Recommendation** field displays the SQL query to create, alter, or replace the index.
+    - **Drop Unused Index**: An **Index** field displays the name of the index to drop; and a **Description** field displays the reason for dropping the index.
+
+[Admin users]({{ link_prefix }}security-reference/authorization.html#admin-role) will see an action button in the final column, which will execute the SQL statement suggested by the schema insight, for example "Create Index". Upon clicking the action button, a confirmation dialog displays a warning about the cost of [online schema changes]({{ link_prefix }}online-schema-changes.html) and the option to copy the SQL statement for later execution in a SQL client.
+
+## Configuration
+
+You can configure the behavior of insights using the following [cluster settings]({{ link_prefix }}cluster-settings.html).
+
+### Workload insights settings
+
+You can configure [**Workload Insights**](#workload-insights-tab) with the following [{{ link_prefix }}cluster settings]({{ link_prefix }}cluster-settings.html):
+
+| Setting                                                                | Default value | Description                                                                                                                                                                                   | Where used                           |
+|------------------------------------------------------------------------|---------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------|
+|`sql.insights.anomaly_detection.enabled`                                | `true`        | Whether or not anomaly insight detection is enabled. When true, CockroachDB checks if [execution latency]({{ link_prefix }}ui-sql-dashboard.html#kv-execution-latency-99th-percentile) was greater than the p99 latency and more than double the median latency. | Statement executions                 |
```

Comment:
For each row I have the following suggestion:
- In the 'Setting' column, the cluster setting should be a link to that setting's anchor on the Cluster Settings page
- The text in the 'Where used' column should be a link directly to the docs for the UI page / section named there

Reply:
Oh and I also suggest doing links to the settings rather than repeating the default values here, since they will become wrong probably surprisingly quickly, and folks can always look up the canonical values using the links to the always-up-to-date version on the cluster settings page